### DISCUSSION TITLE
Integer input example

### DISCUSSION THREAD

**Discussion by timeopochin** [ArthurSonzogni/FTXUI#558] [Show and tell]
Hello!

I just discovered this project, it's wonderful! I've been playing around with it and I think it will be part of many of my future projects. I needed to have a way to input an integer, and I thought my implementation was worth sharing. I'm not sure if it's the sort of thing that would make sense to add to the library which is why I haven't made a pull request; but I could always try to make it part of the project in the future.

Here it is: [FTXUI integer input example](https://gist.github.com/timeopochin/f3187a9a40c3aebc6425c56e10424a98)

It does require C++11 as I use [std::stoi](https://en.cppreference.com/w/cpp/string/basic_string/stol).

_P.S. Actually, maybe it would be a good thing to add to the examples, even if it isn't part of the library itself? Let me know :)_

**Comment by ArthurSonzogni** [comment]
Nice!

I tried, it worked. Maybe we can add a link to it from the wiki?
